client/stylesheets/_toCat.scss

```diff
@@ -5165,16 +5170,25 @@ On click the images will be displayed at normal size to complete the effect
     border-bottom: 1px solid #d3d3d7;
     padding: 30px 0;
     margin: 10px 0;
-    width: 70%;
+    width: 100%;
 }
 
+.profileSettingsLi h4 {
+    font-weight: 400 !important;
+    font-size: 1.1rem !important;
+}
+
+.profileSettingsLi:nth-child(2) ul li:nth-child(2), .profileSettingsLi:nth-child(3) ul li:nth-child(2) {
+    width: 80% !important; 
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Line contains trailing whitespace